controllers/common.go

```diff
@@ -17,12 +17,25 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
+/* #nosec G101 */
```

Comment:
@xrajesh `gosec` flags the variable name "secret" as a security violation.
@dislbenn Can you add this explanation in the code comment?

Reply:
@jlpadilla @xrajesh correct. In sonar, this will mark the secret as a security violation, which will cause the sonar job to fail. This comment will just ignore that as a false positive for us.

Reply:
Added the comment for this flag.